src/priyanka/Assignment22/PrimeNumbers.java

```diff
@@ -0,0 +1,35 @@
+package priyanka.Assignment22;
+
+import java.util.Scanner;
+
+/*Assignment-22 : 18th Nov'2020 
+Print all prime numbers between 2 to 100.
+*/
+public class PrimeNumbers {
+
+	void primeNo(int startNum, int endNum) {
+
+		for (int index = startNum; index <= endNum; index++) {
+			int count = 0;
+			for (int innerIndex = 1; innerIndex <= endNum; innerIndex++) {
+				if (index % innerIndex == 0) {
+					count++;
```

Comment:
once you are in if condition you should break right, bcoz now you knwo this number is not prime number

pls refer this code
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/688/files